distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -4,29 +4,37 @@
 Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
 ====
 
-=== ${branding} Installation Prerequisites
+=== Installation Prerequisites
 
 These are the system/environment requirements to configure _prior_ to an installation.
 
-
 ==== Java Requirements
 
 * Install/Upgrade to Java 8 http://www.oracle.com/technetwork/java/javase/downloads/index.html[J2SE 8 SDK]
 ** The recommended version is http://www.oracle.com/technetwork/java/javase/8u60-relnotes-2620227.html[8u60] or later.
 * Supported platforms are *NIX - Unix/Linux/OSX, Solaris, and Windows.
 * http://www.oracle.com/technetwork/java/javase/downloads/index.html[JDK8] must be installed.
 * The `JAVA_HOME` environment variable must be set to the location where the JDK is installed.
+** Replace `<JAVA_VERSION>` with the the version installed.
 
 .Setting `JAVA_HOME` on *NIX
 ----
-JAVA_HOME=/usr/java/jdk1.8.0
+JAVA_HOME=/usr/java/jdk<JAVA_VERSION>
 export JAVA_HOME
 ----
 
 .Setting `JAVA_HOME` on Windows
-----
-set JAVA_HOME=C:\Program Files\Java\jdk1.8.0
-----
+. Select the *Windows START* Menu.
+. Search *Environment Variables*.
+. Select *System Properties*.
```

Comment:
Even though this is the name of the window, it's not what you see in the search results. Maybe say "Select *Edit the system environment variables*"

Also, if you click *Edit environment variables for your account* it takes you straight it goes straight to the variables and you skip a few steps